README.md

```diff
@@ -28,7 +28,7 @@ Node.js v16 or later.
 ## Installation
 
 ```shell
-npm install govuk-prototype-components --save
+npm install @x-govuk/govuk-prototype-components
```

Comment:
Does `npm` need the prefix? I had thought that the package names were global, even if they were published under an org account?

Reply:
No, the entire string needs to be unique (organisations are confusing on npm). As way of example, these are 2 different packages:

* https://www.npmjs.com/package/vue
* https://www.npmjs.com/package/@astrojs/vue

Reply:
@paulrobertlloyd that’s annoying! I thought that this package was already published under the `x-gov` npm org, but maybe not? https://www.npmjs.com/package/task-list-component

Reply:
Although they're all listed under here already? https://www.npmjs.com/org/x-govuk  so confused!

Reply:
Organisation !== Scope. I think you need to have access to an organisation to publish a package using your organisation name as the scope, but you need to explicitly include the organisation name as the scope in the published package’s name. Confusing, right!?

Reply:
Ah it seems you can choose to either use the org scope or not. https://docs.npmjs.com/about-organization-scopes-and-packages